src/app/asset-upload/asset-upload.component.scss

```diff
@@ -1,9 +1,25 @@
 mat-expansion-panel {
     border: 1px solid #e8e8e8;
+    border-bottom: none !important;
     border-radius: 0;
     box-shadow: none !important;
 }
 
-#fileInput {
+input {
     display: none;
+}
+
+.mat-flat-button.mat-primary, .mat-raised-button.mat-primary, .mat-fab.mat-primary, .mat-mini-fab.mat-primary {
+    background-color: #1876d2;
```

Comment:
We shouldn't need to define these colors explicitly. This should be derived from our application theme.

https://material.angular.io/guide/theming

We can agree to fix this later. For colors not defined in our theme, we should define them as constants in a top level scss file and import them for use to ensure consistency across the app.

Reply:
For now just deleted this from the scss